src/pages/home/sidebar/SidebarScreen/FloatingActionButtonAndPopover.tsx

```diff
@@ -397,6 +412,77 @@ function FloatingActionButtonAndPopover({onHideCreateMenu, onShowCreateMenu}: Fl
         ];
     }, [canUseCombinedTrackSubmit, translate, selfDMReportID, hasSeenTrackTraining, isOffline]);
 
+    const quickActionMenuItems = useMemo(() => {
+        // Define common properties in baseQuickAction
+        const baseQuickAction = {
+            label: translate('quickAction.header'),
+            isLabelHoverable: false,
+            floatRightAvatars: quickActionAvatars,
+            floatRightAvatarSize: CONST.AVATAR_SIZE.SMALL,
+            numberOfLinesDescription: 1,
+            tooltipAnchorAlignment: {
+                vertical: CONST.MODAL.ANCHOR_ORIGIN_VERTICAL.BOTTOM,
+                horizontal: CONST.MODAL.ANCHOR_ORIGIN_HORIZONTAL.LEFT,
+            },
+            tooltipShiftHorizontal: styles.popoverMenuItem.paddingHorizontal,
+            tooltipShiftVertical: styles.popoverMenuItem.paddingVertical / 2,
+            renderTooltipContent: renderQuickActionTooltip,
+            tooltipWrapperStyle: styles.quickActionTooltipWrapper,
+        };
+
+        if (quickAction?.action) {
+            return [
+                {
+                    ...baseQuickAction,
+                    icon: getQuickActionIcon(quickAction?.action),
+                    text: quickActionTitle,
+                    description: !hideQABSubtitle ? ReportUtils.getReportName(quickActionReport) ?? translate('quickAction.updateDestination') : '',
+                    onSelected: () => interceptAnonymousUser(() => navigateToQuickAction()),
+                    shouldShowSubscriptRightAvatar: ReportUtils.isPolicyExpenseChat(quickActionReport),
+                    shouldRenderTooltip: quickAction.isFirstQuickAction,
+                },
+            ];
+        }
+        if (!isEmptyObject(policyChatForActivePolicy)) {
+            return [
+                {
+                    ...baseQuickAction,
+                    icon: Expensicons.ReceiptScan,
+                    text: translate('quickAction.scanReceipt'),
+                    description: ReportUtils.getReportName(policyChatForActivePolicy),
+                    onSelected: () =>
+                        interceptAnonymousUser(() => {
+                            selectOption(() => {
+                                const isValidReport = !(isEmptyObject(policyChatForActivePolicy) || ReportUtils.isArchivedRoom(policyChatForActivePolicy, reportNameValuePairs));
+                                const quickActionReportID = isValidReport ? policyChatForActivePolicy?.reportID ?? '-1' : ReportUtils.generateReportID();
+                                IOU.startMoneyRequest(CONST.IOU.TYPE.SUBMIT, quickActionReportID ?? '-1', CONST.IOU.REQUEST_TYPE.SCAN, true);
```

Comment:
Addition of this caused `You no longer have access to your previous...` to be displayed when first expense was submitted via QAB.